traveler-ios-ui/TravelerKitUI/Views/CalendarDateCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CalendarDateCell.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-10-29.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+class CalendarDateCell: UICollectionViewCell {
+    @IBOutlet weak var dateLabel: UILabel!
+
+    var date: Date? {
```

Comment:
This cell should be way dumber than knowing about dates and stuff.

It should ONLY have a label and selected and deselected states